.github/workflows/testcases.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       - run: python3 manage.py test
       - run: pip3 install coverage
       - run: coverage run ./manage.py test
+      - run: pip install setuptools
```

Comment:
I think we should move this above requirements installation.

Reply:
Sure making the change